viper/modules/office.py

```diff
@@ -292,8 +293,6 @@ def xmlmeta(self, zip_xml):
                 embedded_list.append(name.split('/')[-1])
             if name == 'word/vbaProject.bin':
                 vba_list.append(name.split('/')[-1])
-            if name.startswith('word/activeX/'):
```

Comment:
I believe I'm missing this commit: https://github.com/Rafiot/viper/commit/4f8df20d0e069cf4194e1efae3ec0d0aaae0dee9 not sure how that happened...